typespec-tests/src/main/java/com/_specs_/azure/core/lro/rpc/legacy/LegacyServiceVersion.java

```diff
@@ -4,12 +4,10 @@
 
 package com._specs_.azure.core.lro.rpc.legacy;
 
-import com.azure.core.util.ServiceVersion;
-
 /**
  * Service version of LegacyClient.
  */
-public enum LegacyServiceVersion implements ServiceVersion {
+public enum LegacyServiceVersion implements com.azure.core.util.ServiceVersion {
```

Comment:
Why do we now have fully qualified package name instead of importing it?

Reply:
Will revert.

Reason is one test case got name=`Service` and that result to `ServiceVersion`.